.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
-  - "4.2"
+  - "4"
```

Comment:
Good catch, @t1st3 
> Please note that our LTS "Argon" release line has moved from v4.2.x to v4.3.x due to the security fixes enclosed. There will be no further updates to v4.2.x. Users are advised to upgrade to v4.3.0 as soon as possible.

Is there a way one can specify "always use the latest LTS release" in Travis?